crates/turbopack-image/src/process/mod.rs

```diff
@@ -268,13 +268,10 @@ fn encode_image(image: DynamicImage, format: ImageFormat, quality: u8) -> Result
         }
         #[cfg(feature = "webp")]
         ImageFormat::WebP => {
-            use image::codecs::webp::{WebPEncoder, WebPQuality};
-            WebPEncoder::new_with_quality(&mut buf, WebPQuality::lossy(quality)).write_image(
-                image.as_bytes(),
-                width,
-                height,
-                image.color(),
-            )?;
+            use image::codecs::webp::WebPEncoder;
+            let encoder = WebPEncoder::new_lossless(&mut buf);
```

Comment:
Do we want it to be lossless? Probably not?

Reply:
doesn't seem to other ctor even, if I'm reading correctly https://github.com/image-rs/image/blob/6edf8ae492c4bb1dacb41da88681ea74dab1bab3/src/codecs/webp/encoder.rs#L19

Reply:
that's unfortunate, maybe create a linear ticket to look into some way to get lossy webp back?

Reply:
yeah will do